diff --git a/emscripten/tools/shared.orig.py b/emscripten/tools/shared.py
index 76b2d1c32..4af5dc920 100644
--- a/emscripten/tools/shared.orig.py
+++ b/emscripten/tools/shared.py
@@ -502,14 +502,14 @@ def check_sanity(force=False):
     if sanity_is_correct():
       return
 
-    if os.path.exists(sanity_file):
-      sanity_data = utils.read_file(sanity_file)
-      logger.info('old sanity: %s' % sanity_data)
-      logger.info('new sanity: %s' % expected)
-      logger.info('(Emscripten: config changed, clearing cache)')
-      cache.erase()
-    else:
-      logger.debug(f'sanity file not found: {sanity_file}')
+    # if os.path.exists(sanity_file):
+    #   sanity_data = utils.read_file(sanity_file)
+    #   logger.info('old sanity: %s' % sanity_data)
+    #   logger.info('new sanity: %s' % expected)
+    #   logger.info('(Emscripten: config changed, clearing cache)')
+    #   cache.erase()
+    # else:
+    #   logger.debug(f'sanity file not found: {sanity_file}')
 
     perform_sanity_checks()
 
